calendar/controllers/event.controller.js

```diff
@@ -0,0 +1,114 @@
+const db = require('../models');
+const Event = db.Event;
+const User = db.User;
+const sequelize = db.sequelize;
+const Op = db.Sequelize.Op;
+
+const userAttributes = ['id', 'email', 'firstName', 'lastName'];
+const creator = { model: User, as: 'creator', attributes: userAttributes };
+const participants = {
+	model: User,
+	as: 'participants',
+	attributes: userAttributes,
+	through: {
+		attributes: [],
+	},
+};
+
+module.exports.create = async (req, res) => {
+	try {
+		if (!req.body.title) {
+			res.status(400).send({
+				message: 'Content can not be empty!',
+			});
+			return;
+		}
+		const newEvent = await sequelize.transaction(async (transaction) => {
```

Comment:
I'd suggest separating the code which is responsible for creating events (dealing with databases logic) into separate file eventResource, eventDBResource or smth like that.

In this function, we have an HTTP layer (req, res) as well as DB layer (sequalize). Making this function working only with one layer could make it more abstract and easier to read.